automation/jenkins_build-containers.sh

```diff
@@ -5,6 +5,7 @@ set -ev
 DOCKERFILES=( Dockerfile_yocto-build-env Dockerfile_balena-push-env )
 
 REVISION=$(git rev-parse --short HEAD)
+NAMESPACE=${NAMESPACE:-resin}
```

Comment:
Will `NAMESPACE` be set from the outside? As a part of `s,resin,balena,` we could make it default to `balena` and have the caller overwrite it to `resin`.

Reply:
@mtoman currently we push to https://hub.docker.com/r/resin/resinos, and there are no plans to change this to balena as we are moving away from pushing to dockerhub. So the default is "resin" not to  break things. In the short term this will change to use "balena deploy" instead of "docker push", but at least this change allows to set a NAMESPACE for a personal dockerhub account so that as developers we can test things without pushing to a production repository.

Reply:
makes sense, thanks